lib/services/local/headlessauthn_watcher_test.go

```diff
@@ -191,7 +191,7 @@ func TestHeadlessAuthenticationWatcher(t *testing.T) {
 
 		_, err := s.watcher.Wait(waitCtx, pubUUID, func(ha *types.HeadlessAuthentication) (bool, error) { return true, nil })
 		require.Error(t, err)
-		require.Equal(t, waitCtx.Err(), err)
+		require.Equal(t, context.DeadlineExceeded, trace.Unwrap(err))
```

Comment:
```suggestion
		require.ErrorIs(t, race.Unwrap(err), context.DeadlineExceeded)
```

This should be a bit more resilient to changes. (We might not need the trace.Unwrap, I don't recall now.)

Reply:
:+1:  looks the unwrap is not needed fyi

Reply:
Oh, cool. I think trace does errors.Is/errors.As correctly now, but I'm not sure if it was always the case, so I always end up double-checking.